openedx/features/smart_referral/views.py

```diff
@@ -0,0 +1,23 @@
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
```

Comment:
We have two file for views, this file (`views.py`) and then another file `api_views.py`, also in views.py we have an api view. so let's keep just one either views.py or api_views.py. If I'd me given an option to choose one I'll choose views.py, it's very nice to be more explicit by api_views.py for api view. But in all of our apps we have written api views in `views.py` so just to be consistent with them. edX has used api_views in some apps to separate out api views from other views. but in our case we just have api views.

Reply:
I noticed in our 'Idea' app that we have used api_views to separate rest api views from others and I find it a good approach so I think if edx already is doing this and we have started this so it would be better to continue using api_views which will ease our code navigation.